services/token_update_nft.proto

```diff
@@ -34,19 +34,34 @@ import "google/protobuf/wrappers.proto";
 /**
  * At consensus, updates an already created Non Fungible Token to the given values.
  * 
- * If no value is given for a field, that field is left unchanged. For an immutable tokens (that is,
- * a token without an admin key), only the expiry may be updated. Setting any other field in that
- * case will cause the transaction status to resolve to TOKEN_IS_IMMUTABLE.
+ * If no value is given for a field, that field is left unchanged.
+ * Only certain fields such as metadata can be updated. For an immutable token (that is, a token
+ * without an admin key), only the expiry may be updated. Setting any other field in that case will
+ * cause the transaction status to resolve to TOKEN_IS_IMMUTABLE.
  *
+ * Updating the metadata of an NFT does not affect its ownership or transferability.
+ * This operation is intended for updating attributes of individual NFTs in a collection.
+ *
+ * --- Signing Requirements ---
+ * 1. Whether or not a token has an admin key, its expiry can be extended with only the transaction
+ *    payer's signature.
+ * 2. To update an NFT, the account that owns the NFT must sign the transaction.
+ * 3. The metadata key is not required to be included in the TokenUpdateNftTransactionBody as it is
+ *      not required to set the metadata. The Admin key is required to sign the metadata update.
```

Comment:
It makes sense the metadata key is not included, but is it the Admin key or the metadata key that is required to sign the metadata update?